### PR TITLE
zh-translation/about/faq/security/mysql-with-mtls.md

### PR DESCRIPTION
**Pull request by youhonglian**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/about/faq/security/mysql-with-mtls.md [hunk 1]
```diff
@@ -5,51 +5,10 @@ weight: 95
 keywords: [mysql,mtls]
 ---
 
-您可能会发现安装 Istio 后 MySQL 无法连接。这是因为在 `demo` [安装配置](/zh/docs/setup/additional-setup/config-profiles/)中默认使用的 `PERMISSIVE` 模式不适用于 MySQL。您可能会看到类似于 `ERROR 2013 (HY000): Lost connection to MySQL server at 'reading initial communication packet', system error: 0` 的错误。
+安装 Istio 后，您可能会发现 MySQL 无法连接。 这是因为 MySQL 是 [服务器优先](/zh/docs/ops/deployment/requirements/#server-first-protocols) 协议，
+这会干扰 Istio 的协议检测。 特别是，使用“PERMISSIVE”mTLS 模式可能会导致问题。
+您可能会看到诸如“ERROR 2013 (HY000): Lost connection to MySQL server at
+'读取初始通信包'，系统错误：0`。
```
Comment (rootsongjc) [inline content/zh/about/faq/security/mysql-with-mtls.md:11]: 这段标点符号好像有点问题。

Comment (youhonglian) [reply content/zh/about/faq/security/mysql-with-mtls.md]: resolved

**Review by kebe7jun — APPROVED** [review]

**Comment by youhonglian** [comment]
@googlebot I fixed it

**Pull request merged**